<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="ViewportFrame">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="L_MainLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="M_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="A_New"/>
    <addaction name="A_Open"/>
    <addaction name="A_Save"/>
    <addaction name="A_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="A_Quit"/>
   </widget>
   <widget class="QMenu" name="M_Edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="A_Delete"/>
    <addaction name="A_Copy"/>
    <addaction name="A_Cut"/>
    <addaction name="A_Paste"/>
    <addaction name="separator"/>
    <addaction name="A_Undo"/>
    <addaction name="A_Redo"/>
   </widget>
   <widget class="QMenu" name="M_Help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="A_AboutCreator"/>
    <addaction name="A_AboutGame"/>
    <addaction name="A_AboutQt"/>
   </widget>
   <addaction name="M_File"/>
   <addaction name="M_Edit"/>
   <addaction name="M_Help"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QToolBar" name="T_File">
   <property name="windowTitle">
    <string>File toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="A_New"/>
   <addaction name="A_Open"/>
   <addaction name="A_Save"/>
  </widget>
  <widget class="QToolBar" name="T_Edit">
   <property name="windowTitle">
    <string>Edit toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="A_Delete"/>
   <addaction name="A_Copy"/>
   <addaction name="A_Cut"/>
   <addaction name="A_Paste"/>
  </widget>
  <widget class="QToolBar" name="T_UndoRedo">
   <property name="windowTitle">
    <string>Undo/Redo toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="A_Undo"/>
   <addaction name="A_Redo"/>
  </widget>
  <widget class="QDockWidget" name="D_Attributes">
   <property name="windowTitle">
    <string>Attributes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="DW_AttributeTable">
    <layout class="QGridLayout" name="DWL_AttributeTable">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="A_New">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>Start the creation of a new level.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="A_Open">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Edit the chosen level file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="A_Save">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the level currently being edited.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="A_SaveAs">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save this level to a different file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="A_Copy">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy currently selected level element.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="A_Cut">
   <property name="icon">
    <iconset theme="edit-cut"/>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Move the currently selected level element.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="A_Paste">
   <property name="icon">
    <iconset theme="edit-paste"/>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste the copied level element.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="A_Undo">
   <property name="icon">
    <iconset theme="edit-undo"/>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo last edition.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="A_Redo">
   <property name="icon">
    <iconset theme="edit-redo"/>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo last undo.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="A_Delete">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected level element</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="A_AboutCreator">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About WorldStoryCreator...</string>
   </property>
  </action>
  <action name="A_AboutGame">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About WorldStory...</string>
   </property>
  </action>
  <action name="A_AboutQt">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="A_Quit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Creator.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
